src/Proxy.NET/Endpoints/Attendee.cs

```diff
@@ -0,0 +1,60 @@
+using System.Net;
+using System.Text.Json;
+using Proxy.NET.Models;
+using Proxy.NET.Services;
+
+namespace Proxy.NET.Endpoints;
+
+public static class Attendee
+{
+    private enum RequestType
+    {
+        AttendeeAdd,
+        AttendeeGetKey
+    }
+
+    public static void AttendeeEndpoints(this IEndpointRouteBuilder routes)
+    {
+        MapRoute.Post(routes, RequestType.AttendeeAdd, AttendeeAdd, Auth.Type.Jwt, "/attendee/event/{event_id}/register");
+        MapRoute.Get(routes, RequestType.AttendeeGetKey, AttendeeGetKey, Auth.Type.Jwt, "/attendee/event/{event_id}");
+    }
+
+    /// <summary>
+    /// Represents an asynchronous operation that can return a value.
+    /// </summary>
+    /// <returns>A task representing the asynchronous operation.</returns>
+    private static async Task AttendeeAdd(HttpContext context, RequestType requestType, string extPath)
+    {
+        var attendeeService = context.RequestServices.GetRequiredService<IAttendeeService>();
+        if (context.Items["RequestContext"] is not string userId || string.IsNullOrEmpty(userId))
+            throw new ArgumentException("Request context not found");
```

Comment:
This seems like it'd be better a [middleware](https://learn.microsoft.com/en-us/aspnet/core/fundamentals/minimal-apis/middleware?view=aspnetcore-8.0) that you then would add to the route (or add globally).